mmdet3d/models/voxel_encoders/voxel_encoder.py

```diff
@@ -486,3 +486,130 @@ def fusion_with_mask(self, features, mask, voxel_feats, coors, img_feats,
         out = torch.max(voxel_canvas, dim=1)[0]
 
         return out
+
+
+@MODELS.register_module()
+class CylindricalVFE(nn.Module):
+    """Cylindrical Voxel feature encoder used in Cylinder3D.
+
+    It encodes features of voxels and their points. It could also fuse
+    image feature into voxel features in a point-wise manner.
+    The number of points inside the voxel varies.
+
+    Args:
+        in_channels (int, optional): Input channels of VFE. Defaults to 4.
+        feat_channels (list(int), optional): Channels of features in VFE.
+        with_distance (bool, optional): Whether to use the L2 distance of
+            points to the origin point. Defaults to False.
+        with_cluster_center (bool, optional): Whether to use the distance
```

Comment:
Mind the docstring about input arguments. They are not matched with input arguments.